admin/commands/list_test.go

```diff
@@ -12,6 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//nolint:lll
```

Comment:
I guess we can disable this linter for tests globally

Reply:
A good idea, thanks, updated.